var/spack/repos/builtin/packages/xsdk/package.py

```diff
@@ -375,46 +347,54 @@ class Xsdk(BundlePackage, CudaPackage, ROCmPackage):
     xsdk_depends_on("phist kernel_lib=tpetra", when="+trilinos +phist")
     xsdk_depends_on("phist kernel_lib=petsc", when="~trilinos +phist")
     xsdk_depends_on("phist@develop ~fortran ~scamac ~openmp ~host ~int64", when="@develop +phist")
+    xsdk_depends_on("phist@1.11.2 ~fortran ~scamac ~openmp ~host ~int64", when="@0.8.0 +phist")
     xsdk_depends_on("phist@1.9.5 ~fortran ~scamac ~openmp ~host ~int64", when="@0.7.0 +phist")
     xsdk_depends_on("phist@1.9.3 ~fortran ~scamac ~openmp ~host ~int64", when="@0.6.0 +phist")
-    xsdk_depends_on("phist@1.8.0 ~fortran ~scamac ~openmp ~host ~int64", when="@0.5.0 +phist")
-    xsdk_depends_on("phist@1.7.5 ~fortran ~scamac ~openmp ~host ~int64", when="@0.4.0 +phist")
 
     xsdk_depends_on(
-        "ginkgo@develop ~openmp", when="@develop +ginkgo", cuda_var="cuda", rocm_var="rocm"
+        "ginkgo@develop +mpi ~openmp", when="@develop +ginkgo", cuda_var="cuda", rocm_var="rocm"
```

Comment:
The [deal.II pipeline failure](https://gitlab.spack.io/spack/spack/-/jobs/4282126#L961) seems to be triggered by the `+mpi` and the system that deal.II uses for dependencies which was not updated for this setting.

I'm checking if there's a ginkgo fix possible but that looks unlikely. For a (hopefully) successful build, mpi can be removed here.

```suggestion
        "ginkgo@develop ~openmp", when="@develop +ginkgo", cuda_var="cuda", rocm_var="rocm"
```

Reply:
Perhaps CI using wrong ginkgo build (~mpi) from cache with dealii - hence this failure?

cc: @scottwittenburg

Reply:
Perhaps I should separate out the ginkgo PR - to see if the failure is related to xdk code [or not]